src/Cli/dotnet/ToolPackage/IToolPackageDownloader.cs

```diff
@@ -16,5 +16,11 @@ internal interface IToolPackageDownloader
             bool isGlobalTool = false,
             bool isGlobalToolRollForward = false
         );
+
+        NuGetVersion GetNuGetVersion(
+            PackageLocation packageLocation, PackageId packageId,
```

Comment:
```suggestion
            PackageLocation packageLocation,
            PackageId packageId,
```
nit, this should probably be on its own line to match the rest of the code

Reply:
Done!